### PR TITLE
FE: Show the right Alarm Message on the Node Status Page.  Right now it says CLEAR.

### PR DESCRIPTION
**Pull request by atif1019**
## Description
On the Node Status Page, Recent Alert says "CLEAR". I think it is not using the right field.
Make the following updates to the front end:

For the first column “Alert Type”, display the Alarm uei field (and should be sortable on that field)

Add a second column named “Count”, this should display the Alarm counter field

Rename the “TIME” column to “Last Updated”. It should be sortable on lastUpdatedTimeMs 

Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2683

### CODE REVIEW

File: ui/src/components/NodeStatus/NodeAlerts.vue [hunk 1]
```diff
@@ -133,9 +151,10 @@ const alertsData = ref<IAlert[]>([])
 const expandedIds = ref(new Set<number>())
 
 const columns = [
-  { id: 'type', label: 'Alert Type' },
+  { id: 'eventUei', label: 'Alert Type' },
```
Comment (Shahbaz-dataq) [inline ui/src/components/NodeStatus/NodeAlerts.vue:154]: remane this lable 'UEI'

**Review by Shahbaz-dataq — APPROVED** [review]
Small change. Otherwise LGTM

File: ui/src/components/NodeStatus/NodeAlerts.vue [hunk 2]
```diff
@@ -39,15 +39,29 @@
                 data-test="data-item"
               >
                 <td class="alert-details-wrapper">
-                  <div class="name headline alert-type" data-test="name">{{ alert?.type || alert?.label || 'Unknown' }}</div>
+                    <div class="name headline alert-type" data-test="name">{{ alert?.uei?.slice(0, 20)+"..." || alert.label || 'Unknown' }}
```
Comment (synqotik) [inline ui/src/components/NodeStatus/NodeAlerts.vue:42]: If `alert.uei` is <= 20 characters, don't display the `...`. May want to change this to a function.

**Review by synqotik — CHANGES_REQUESTED** [review]
Added a comment.

**Review by synqotik — APPROVED** [review]
LGTM

**Pull request merged**